docker/config/never_on_a_server.env

```diff
@@ -4,6 +4,10 @@
 
 # NOTE: These shouldn't be used on -dev, -stage, or -prod environments.
 
+# Flag indicating we're running things in the local development environment
+LOCAL_DEV_ENV=True
```

Comment:
We put this in this file to make sure @milescrabill doesn't accidentally copypasta it to our new server environment. He and I agreed that this was the way to reduce that risk.